src/main/java/org/aerogear/digger/client/services/JobService.java

```diff
@@ -31,10 +37,13 @@
 public class JobService {
 
     private final static String GIT_REPO_URL = "GIT_REPO_URL";
+    private final static String GIT_CREDENTIALS_ID = "GIT_CREDENTIALS_ID";
     private final static String GIT_REPO_BRANCH = "GIT_REPO_BRANCH";
     private final static String BUILD_PARAMETERS = "BUILD_PARAMETERS";
     private static final String JOB_TEMPLATE_PATH = "templates/job.xml";
 
+    private static final Logger LOG = LoggerFactory.getLogger(JobService.class);
```

Comment:
Static reference to slf4j should be avoided: https://wiki.apache.org/commons/Logging/StaticLog